templates/individualModuleInfo.html

```diff
@@ -94,40 +61,43 @@ <h1 class="text-center"><b>Module Info for <u>$aySem</u></b></h1>
-                        </tr>
-                </tbody>
-            </table>
+            <div class="row text-center well">
+                <!--module button links-->
+                <h1><b>More Actions</b></h1>
+                <div class="col-md-6 text-center">
+                    <form id="edit-module-button" name="edit-module-button" action="/editModule" method="post">
+                        <input type="hidden" name="status" value="load">
+                        <input type="hidden" name="code" value="$moduleCode">
+                        <input class="btn btn-lg btn-primary" type="submit" value="Edit General Module Info" data-toggle="tooltip" data-placement="right" title="Edit the module's name, description and MCs">
+                    </form>
+                    $if isFutureAY is True:
+                        <form id="edit-mounting-button" name="edit-mounting-button" action="/editMounting" method="post">
+                            <input type="hidden" name="status" value="load">
+                            <input type="hidden" name="code" value="$moduleCode">
+                            <input type="hidden" name="aySem" value="$aySem">
+                            <input type="hidden" name="mountingValue" value="$mountingStatus">
+                            <input type="hidden" name="quota" value="$quota">
+                            <input class="btn btn-lg btn-primary" type="submit" value="Edit Specific Module Info" data-toggle="tooltip" data-placement="right" title="Edit the module's mounting, prerequisites/preclusions and quota">
+                        </form>
+                    $if True:
+                        <form id="view-module-button" action="/viewModule"  method="get">
+                            <input type="hidden" name="code" value="$moduleCode">
+                            <input class="btn btn-lg btn-primary" type="submit" value="Back to Overview">
+                        </form>
+                </div>
+                <div class="col-md-6 text-center">
+                    <form id="view-students-planning-to-take-module" name="view-students-planning-to-take-module" action="/studentsAffectedByModule" method="get">
+                        <input type="hidden" name="code" value="$moduleCode">
+                        <input type="hidden" name="aysem" value="$aySem">
+                        <input type="submit" class="btn btn-lg btn-primary" value="View students that will be affected by this module">
+                    </form>
+                    <form id="view-overlapping-with-module" name="view-overlapping-with-module" action="/overlappingWithModule" method="get">
+                        <input type="hidden" name="code" value="$moduleCode">
+                        <input type="hidden" name="aysem" value="$aySem">
+                        <input type="submit" class="btn btn-lg btn-primary" value="View modules overlapping with this module">
```

Comment:
Additionally, "More Actions" should NOT qualify as a <code>h1</code> element since this section is a part of the module page. Consider using 1-tier lower, which is <code>h2</code>.